models/business-unit/src/associate/transformers.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  type TReference,
+  type TReferenceGraphql,
+  Reference,
+} from '@commercetools-test-data/commons';
+import { Transformer } from '@commercetools-test-data/core';
+import type {
+  TAssociateDefault,
+  TAssociateRest,
+  TAssociateGraphql,
+} from './types';
+
+const transformers = {
+  default: Transformer<TAssociateDefault, TAssociateDefault>('default', {
+    buildFields: ['associateRoleAssignments', 'customer'],
+  }),
+  rest: Transformer<TAssociateDefault, TAssociateRest>('rest', {
+    buildFields: ['associateRoleAssignments', 'customer'],
+    replaceFields: ({ fields }) => {
+      const customer: TReference<'customer'> =
+        Reference.presets.customerReference
+          .customerReference()
+          .id(fields.customer.id)
+          .build();
+      return {
+        ...fields,
+        customer,
+      };
+    },
+  }),
+  graphql: Transformer<TAssociateDefault, TAssociateGraphql>('graphql', {
+    buildFields: ['associateRoleAssignments', 'customer'],
+    replaceFields: ({ fields }) => {
+      const customerRef: TReferenceGraphql = Reference.presets.customerReference
+        .customerReference()
```

Comment:
Is this really the syntax? `customerReference` repeated twice??

Reply:
Looks like this is in fact [legal syntax](https://github.com/commercetools/test-data/blob/main/models/commons/src/reference/presets/associate-role/index.ts#L3), should probably change this

Reply:
Some presets are in directories and exported this way, and others are all grouped at the top level

Reply:
If we do change it, make sure to change it everywhere it's used
![Screenshot 2024-02-13 at 1 34 30 PM](https://github.com/commercetools/test-data/assets/87667330/6dba9495-4d30-4582-9dd6-30f77f2a64bf)

Reply:
Made a ticket for this 😎